flowzone.yml

```diff
@@ -457,6 +513,22 @@ jobs:
     steps:
       - *logGitHubContext
 
+      # https://stackoverflow.com/a/70249520/1559300
+      - name: Check secrets
```

Comment:
I think this got moved to the wrong job during a rebase. I would expect this step to be part of sanitize inputs.